docs/formulary/lengths.rst

```diff
@@ -4,4 +4,15 @@
 Length Plasma Parameters (`plasmapy.formulary.lengths`)
 *******************************************************
 
+.. currentmodule:: plasmapy.formulary.lengths
+
 .. automodapi:: plasmapy.formulary.lengths
+
+Examples
+========
+
+.. nbgallery::
+    :caption: Examples
+
+    /notebooks/formulary/ExB_drift
+    /notebooks/formulary/iter
```

Comment:
You would add the `magnetosphere` notebook here.

```suggestion
    /notebooks/formulary/ExB_drift
    /notebooks/formulary/iter
    /notebooks/formulary/magnetosphere
```

Reply:
Good catch!  I must have been spacing out on that one.